docssrc/src/argument_list.md

```diff
@@ -57,6 +57,20 @@ $$\downarrow$$
 
 $$\downarrow$$
 
+> ### Allowing any value (Optional)
+>
+> If you want to do your own validation you can enable the `allowAnyValue` method to set whether any value is allowed. By default, this is disabled.
+>
+> When `allowAnyValue` is enabled, validation is skipped, and it is expected you will manually perform any validation required:
```

Comment:
```suggestion
> When `allowAnyValue` is set to `true`, list validation is skipped, and any value a player enters will be allowed in the list regardless of the list provided for the argument. Note that duplicate values are still only permitted when `allowDuplicates` is set to `true` too:
```
Maybe this is a bit better?

Reply:
I don't particularly like the phrase "list validation", we could just be clear as day and remove it. By saying "... is set to `true`, any value a player enters ...", it's so much cleaer.